lib/landing_session.js

```diff
@@ -227,10 +243,20 @@ class LandingSession extends Session {
       willBeLanded = `${head}...${willBeLanded}`;
     }
 
-    cli.log('To finish landing:');
-    cli.log(`1. Run \`git push ${upstream} ${branch}\``);
-    const url = `https://github.com/${owner}/${repo}/pull/${prid}`;
-    cli.log(`2. Post in ${url}: \`Landed in ${willBeLanded}\``);
+    const pushCommandArgs = ['push', upstream, branch];
+    const prUrl = `https://github.com/${owner}/${repo}/pull/${prid}`;
+    const landComment = `Post in ${prUrl}: \`Landed in ${willBeLanded}\``;
+    const autoLand = await asyncQuestion(prid);
+    if (autoLand) {
+      await runAsync('git', pushCommandArgs);
```

Comment:
If I remember correctly...a failed `git push` child process should exit with 1 and make this process also exit with 1 (through runAsync)?

Reply:
AFAICT `runAsync(x, y, { ignoreFailure:false })` will exit the process.